### ISSUE TITLE
[Feature] Allow traceviewer to load traces from other origins

### ISSUE THREAD

**Issue by vovsemenv** [microsoft/playwright#21382]
At https://trace.playwright.dev/ we have query parameter `trace` to specify url of trace to load.
We want to add open our traces in this domain by link like that `https://trace.playwright.dev/?trace=https://example.org` but get cors error 
```
Access to fetch at 'https://example.org/' from origin 'https://trace.playwright.dev' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

Maybe it is possible to load the trace with `no-cors` option when we add `?no-cors=true` query parameter?

**Comment by pavelfeldman** [comment]
TraceViewer already allows loading tracing from other origins using cors. Not sure what you mean by "load trace with no-cors", that's not how the web works.

**Comment by pavelfeldman** [comment]
Closing as per above, please feel free to open a new issue if this does not cover your use case.
